en/firefox_privacy_notice.md

```diff
@@ -1,152 +1,131 @@
+### Review Checker {: #review-checker }
 
-* __Technical data__: To display which devices are synced to your Firefox Account and for security functionality, we store your device operating system, browser and version, timestamp, locale, and the same information for devices connected to your account. If you use your Firefox Account to log into other websites or services (such as AMO or Pocket), we receive the timestamp of those log-ins.
+Users in the US can opt in to Review Checker, our guide to product reviews powered by Mozilla’s [Fakespot](https://www.fakespot.com/), which helps you understand how reliable product reviews are on [certain shopping sites](https://support.mozilla.org/kb/review-checker-review-quality). 
 
-Read the full documentation or learn more, including how to manage your Firefox Account data or our data practices for [websites and email](https://www.mozilla.org/privacy/websites/).  You can also read the privacy notices for our Firefox Account connected services, which are:
-
-* [Firefox Monitor](https://www.mozilla.org/privacy/firefox-monitor)
-* [Firefox Notes](https://addons.mozilla.org/firefox/addon/notes-by-firefox/)
-* [Pocket](https://getpocket.com/privacy/)
-* [Firefox Sync](https://www.mozilla.org/privacy/firefox/#sync)
-* [Firefox Relay](https://www.mozilla.org/privacy/firefox-relay/)
-* [Mozilla VPN](https://www.mozilla.org/privacy/mozilla-vpn/)
+* __Technical and interaction data__: We use Oblivious HTTP ([OHTTP](https://datatracker.ietf.org/doc/draft-ietf-ohai-ohttp/)) with our partner [Fastly](https://www.fastly.com/privacy/) to provide the service, while ensuring that any data identifying products, URLs, and request payloads are unlinkable to you. For more information about Review Checker, including what websiteshow to turn it off, you can read the [documentation](https://support.mozilla.org/kb/review-checker-review-quality). 
 
 ### Sync {: #sync }
 
-* __Synced data__: If you enable Sync, Mozilla receives the information that you sync across devices in encrypted form. This may include Firefox tabs, add-ons, passwords, payment autofill information, bookmarks, history, and preferences.  Deleting your Firefox Account will delete related Firefox Sync content. You can also read the [documentation](https://moz-services-docs.readthedocs.io/en/latest/sync/).
+* __Synced data__: If you enable sync, Mozilla receives the information that you sync across devices in encrypted form. This may include Firefox tabs, add-ons, passwords, payment autofill information, bookmarks, history, and preferences. Deleting your Mozilla account will delete related Firefox synchronized content. You can also read the [documentation](https://moz-services-docs.readthedocs.io/en/latest/sync/).
```

Comment:
Open question here: payment autofill -> form autofill, given we store addresses as well?

Reply:
Correct, thanks for checking